js/mcqModel.js

```diff
@@ -168,10 +168,59 @@ define([
                 _selectedItems: [],
                 _isAtLeastOneCorrectSelection: false
             });
+        },
+
+        getInteractionObject: function() {
+            var interactions = {
+                correctResponsesPattern: [],
+                choices: []
+            };
+
+            interactions.choices = this.get('_items').map(function(item) {
+                return {
+                    id: (item._index + 1).toString(),
+                    description: item.text
```

Comment:
Hi @oliverfoster, I actually have the xAPI plugin handling this, i.e. where it sees a `choices` object it checks if the description contains a string or an object.  If it's a string it maps it accordingly.

I was actually on the fence about it, but if it's acceptable as is -- i.e. plugin neutral -- I'm ok with it.